kernel/posix/pthread_rwlock.c

```diff
@@ -254,34 +254,4 @@ static u32_t write_lock_acquire(pthread_rwlock_t *rwlock, s32_t timeout)
 	return ret;
 }
 
-static s64_t calculate_timeout(const struct timespec *abstime)
```

Comment:
Hah.  This code wasn't original, you just moved it.  So consider my notes above a suggestion. Still would be good to fix this thing.